contracts/pair_stable_owner/examples/pair_stable_schema.rs

```diff
@@ -0,0 +1,40 @@
+use std::env::current_dir;
```

Comment:
Rename this file `pair_stable_owner_schema.rs`

Reply:
Renamed